content/01_WhyRx.md

```diff
@@ -1,241 +1,126 @@
-			<!--C# Linq pocket reference Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=0596519249&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-		</div>
-
-		<div style="display:inline-block; vertical-align: top; margin: 10px; width: 140px; font-size: 11px; text-align: center">
-			<!--CLR via C# v4 Amazon.co.uk-->
-			<iframe src="http://rcm-uk.amazon.co.uk/e/cm?t=int0b-21&amp;o=2&amp;p=8&amp;l=as1&amp;asins=B00AA36R4U&amp;ref=qf_sp_asin_til&amp;fc1=000000&amp;IS2=1&amp;lt1=_blank&amp;m=amazon&amp;lc1=0000FF&amp;bc1=000000&amp;bg1=FFFFFF&amp;f=ifr" 
-					style="width:120px;height:240px;margin: 10px" 
-					scrolling="no" marginwidth="0" marginheight="0" frameborder="0"></iframe>
-
-		</div>      
-	</div>
</div>
+---
+title : Why Rx?
+---
+
+# PART 1 - Getting started
+
+Rx is a .NET library for processing event streams. Why might you want that?
+
+## Why Rx?
+
+Users want timely information. If you're expecting a parcel, live reports of the delivery van's progress give you more freedom than a suspect 2 hour delivery window. Financial applications depend on continuous streams of up-to-date data. We expect our phones and computers to provide us with all sorts of important notifications. And some applications simply can't work without live information—online collaboration tools and multiplayer games absolutely depend on the rapid distribution and delivery of data.
+
+In short, our systems need to react when interesting things happen.
+
+Live information streams are a basic, ubiquitous element of computer systems. Despite this, they are often a second class citizen in programming languages. Most languages support sequences of data through something like an array, which presumes that the data is sitting in memory ready for our code to read at its leisure. If your application deals with events, arrays might work for historical data, but they aren't a good way to represent events that occur while the application is running. And although streamed data is a pretty venerable concept in computing, it tends to be clunky, with the abstractions often surfaced through APIs that are poorly integrated with our programming language's type system.
+
+This is bad. Live data is critical to a wide range of applications. It should be as easy to work with as lists, dictionaries, and other collections.
+
+The [Reactive Extensions for .NET](https://github.com/dotnet/reactive) (Rx.NET or Rx for short) elevate live data sources to first class citizens. Rx does not require any special programming language support. It exploits .NET's type system to represent streams of data in a way that .NET languages such as C#, F#, and VB.NET can all work with as naturally as they use collection types.
```

Comment:
I'm also wondering if you need to mention `System.Reactive` in this sentence too.. I notice it's referenced later on, but what it is and how it related hasn't really been explained.